my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
```

Comment:
Another possibility would be to define the desired columns in a list, and iterate the plotting over the chosen columns, e.g.:

```python
column_indices = [20, 200, 800]
for col_idx in column_indices:
    col = data[col_idx]
    # <now plot>